pages/docsprint/guides/participation.md

```diff
@@ -4,4 +4,62 @@ permalink: /docsprint/guides/participation/
 
 # Doc Sprint Participation Guide
 
-Here will be a guide to participating in a doc sprint.
+A well-organized doc sprint is designed to improve team documentation quickly and efficiently. Making sure 18F's documentation is accessible and comprehensive is critical for scaling what we do.  
+
+Documentation includes, but is not limited to:
+
+* READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
+* Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
+* Updating our [18F Guides](http://18f.github.io/guides/)
+
+**Here are some helpful materials and resources for participating in a doc sprint.**
+
+## <a name="todo"></a>Event Specific To-Do Lists
+
+Specific information and to-do list for each doc sprint:
+
+* [April 17 Onboarding Doc Sprint](../../onboarding/)
```

Comment:
@melodykramer Going to try to work with project/tech leads to figure out how they're collecting/burning down tasks, and to point them at the Sprint form to register activity.

Regarding the Guides, we're copying the template from the Agile guide. Most of the existing Guides are ported; just a few left. Plus, @nickbristow had a couple comments on accessibility to work in. ('Twould be nice to make the actual style files available as a--dare I say it?--_submodule_...) ;-)